eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/parameters/InputParameters.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.dbsnp.parameters;
+
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersBuilder;
+
+
+public class InputParameters {
+
+    private String fasta;
+
+    private String assemblyReportUrl;
+
+    private String assemblyAccession;
+
+    private String assembly;
```

Comment:
Maybe this could be called `assemblyName` to make clear what is it? I was confused for a moment until I understood it was the name from which the hash for the materialized views is calculated.